### ISSUE TITLE
Git mergetool failed with exitcode 128.

### ISSUE THREAD

**Issue by Obehi** [zawys/vscode-as-git-mergetool#2]
## Steps to reproduce

1. Install git mergetool
2. Click start  git mergetool from the git panel
3. 
4. 

## Expected behavior
Open merge tool without error message

## [Optional] screenshots
![image](https://user-images.githubusercontent.com/9110206/94790581-c224ac00-03d6-11eb-9441-d9a78a93b5b2.png)

![image](https://user-images.githubusercontent.com/9110206/94790533-b507bd00-03d6-11eb-8c97-ab560a43180e.png)

## Environment

- Version of the extension:  v0.10.0
- Operating system: MacOS Catalina V10.15.6
- VS Code version [use Code command “Help: About”]: 1.49.2
- Other installed extensions (if applicable): GitLens, Git Graph

**Comment by zawys** [comment]
What happens when you open a terminal, `cd` to the root of the affected repository, and manually run the command `git mergetool`?

**Comment by Obehi** [comment]
![image](https://user-images.githubusercontent.com/9110206/94802202-847c4f00-03e7-11eb-9803-56b5f75eec09.png)

Ive never tried using mergetool from terminal, but looks like its working?

**Comment by zawys** [comment]
I agree. Those warnings look weird but probably have nothing to do with `git mergetool` and are probably a pipe reuse issue with VS Code / NodeJS.

Would it be possible for you to compress the repository in the failing state and email it to: pj94ye@runbox.com ? 

I am working on version 1.0 of the extension which shall make it possible to avoid `git mergetool` but that might still take some days until I can publish it.

**Comment by zawys** [comment]
When you run `git mergetool` from terminal, is the merge conflict opened in VS Code?

**Comment by Obehi** [comment]
Sorry, its the repository for my work, so my boss would kinda hate me if I shared it. But I tested it on a new repo and it works.
The merge conflict is opened in vscode yea.  
Either way, I am more than happy to wait for the v1.0, so no worries :)

**Comment by jmevel** [comment]
I'm facing the same issue.

I tried to configure the mergetool in my git settings but not sure if this is correct since the documentation doesn't mention this part (I was using KDiff3 before so I let the config I was using in my example)

I don't use VS Code for C# programming (I use Visual Studio) and I always use git from the command line.
After performing a `git merge develop` on my current branch I had conflicts so I ran `code .` to open the current directory in VS Code then ran the command `Start git mergetool` from the palette and got the error mentionned in this ticket.

Here is my `.gitconfig`

```
[merge]
	tool = VSCode
[mergetool]
	keepTemporaries = false
	keepBackup = false
[mergetool "kdiff3"]
	path = C:/Program Files/KDiff3/kdiff3.exe
	trustExitCode = false
[mergetool "VSCode"]
	path = C:/Users/username/AppData/Local/Programs/Microsoft VS Code/Code.exe"
	trustExitCode = false
```

Could you please provide us a more detailed `Installation` section in the Readme?

Thank you

### EDIT:

Actually running the `git mergetool` command directly in the terminal told me the problem: my `.gitconfig` file was invalid because of an extra `"` I forgot to delete.

Now I've another issue is that running the `Start git mergetool` command opens a terminal and ask me to launch the diff merge with **Tortoise Merge**... I need to figure that out because I never configured this tool in my `.gitconfig`
